src/runtime/generator.cpp

```diff
@@ -407,7 +407,7 @@ extern "C" void generatorGCHandler(GCVisitor* v, Box* b) {
 
     if (g->running) {
         v->visitPotentialRange((void**)&g->returnContext,
-                               ((void**)&g->returnContext) + sizeof(g->returnContext) / sizeof(void*));
+                               ((void**)&g->returnContext) + sizeof(*g->returnContext) / sizeof(void*));
```

Comment:
Oh good catch!
I created #746.
Feels a little bit scary that this didn't show up.